### ISSUE TITLE
Usability improvements for "Edit with Oxygen" in the top admin bar

### ISSUE THREAD

**Issue by porg** [soflyy/oxygen-bugs-and-features#2494]
![Oxygen Builder - Single menu item EDIT IN OXYGEN nested in parent menu OXYGEN](https://user-images.githubusercontent.com/737143/139318365-3adfe54e-dda3-4122-9b2f-1cdc8a780f6e.png)

# Proposed Improvements
1. If "Edit with Oxygen" is the sole content of the menu container "Oxygen" then it makes no sense to display a container at all. It's just a wate of clicks each time.
    - If that container can host more options, then it should be intelligent
       - If 2+ entries --> Wrap entries into "Oxygen" menu
       - If only 1 entry --> Show that entry directly in the top admin bar
         - Optimally not in the currently quite long form "Edit with Oxygen"
           -  but instead Oxygen logo + "Edit" and the full "Edit with Oxygen" only in the proper HTML accessibility attributes (though I doubt blind people use a visual page builder)
           -  if only plain text is possible then something like "oxy.Edit" or "Edit w. Oxy"
2. Keyboard shortcut in read mode to trigger "Edit in Oxygen", i.e. simply "E" with certain modifier keys.
    - Should be customizable for conflict resolving/avoidance.
    - This spares you many mouse kilometers 🙂 Some other CMS have this, i.e. Wikis.
    - I don't know whether key binding in read mode would still be in the realm of Oxygen or rather Wordpress.
      - If only do-able as a Wordpress plugin: Would be cool if the Oxygen team made it (didn't find any so far) and utilizes it for read mode keybindings useful in use with Oxygen.
3. #513

**Comment by Flipback8** [comment]
I agree. I would remove the nested edit with Oxygen to Oxygen logo + "Edit".
I would also add "Templates" to the menu where all Oxygen templates are nested in the menu.


 Oxygen templates nested under that menu.

**Comment by porg** [comment]
- Oxygen logo + "Edit"
  - My preferred variant too. Provided the short text variants only for the case that WP may restricts 3rd parties.
- Oxygen templates entry in the admin bar?
  - I don't agree. This bloats the menu a non-primary function.
  - Templates are already quite accessible
    - From the editor itself when selecting i.e. a header, you can jumpt to the corresponding template
    - From the backend in the menu > Oxygen > Templates
  - For wild admin bar customizations there are many plugins, i.e. https://wordpress.org/plugins/toolbar-extras/

**Comment by Flipback8** [comment]
I would like something like this: [Menu](https://snipboard.io/HozYE1.jpg)
![image](https://i.snipboard.io/HozYE1.jpg)

**Comment by Flipback8** [comment]
This was shared in the Oxygen Group. [WPDevDesign – Oxygen – Navigator](https://wordpress.org/plugins/wpdevdesign-oxygen-navigator)
